first-task.css

```
@@ -203,4 +204,66 @@ li {
 .author-text {
   font-family: 'Plus Jakarta Sans';
   font-size: 13px;
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: #232536;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: space-between;
+  padding: 15px 15px;
```

```suggestion
  padding: 15px;
```
Not really necessary to type the same number twice.
If you pass 1 parameter it applies to all the sides
2 params - top/bottom, right/left
3 params - top, right/left, bottom
4 params - top, right, bottom, left